File: JavaOOP/app/src/main/java/com/vkt4u9999/javaoop/MainActivity.java
package com.vkt4u9999.javaoop;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;
import android.widget.ArrayAdapter;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Movable movable = new Cat();
        movable.catGrow();

        Animal cat1= new Cat();
        cat1.catGrow();

    }
}

File: MDevel2/app/src/main/java/com/vkt4u9999/mdevel/MainActivity.kt
package com.vkt4u9999.mdevel

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


//
//        val weatherStation= WeatherStation()
//        weatherStation.addMan(Man("Aleksandra"))
//        weatherStation.addMan(Man("Igor"))
//        weatherStation.addMan(Man("Dima"))
//        weatherStation.updateWeather()

        val configuration= ConfigurationBuilder()
            .color(2)
            .latency(1000)
            .build()

        Log.d("TAG", configuration.toString())
    }
}